src/components/catalog/catalog-element.ts

```diff
@@ -0,0 +1,77 @@
+import { combineLatest, map } from 'rxjs'
+import { html, TemplateResult } from 'lit'
+import { customElement, state } from 'lit/decorators'
+import { layersService } from '../../services/layers/layers.service'
+import { ThemeNodeModel } from '../../services/themes/themes.model'
+import { themesService } from '../../services/themes/themes.service'
+import { mapState } from '../../states/map/map.state'
```

Comment:
I might have kept `state` instead of `states`, but this does not seem of big importance :-)

Reply:
I kept states/ plural for the directory like we did for services/, mixins/, components/.